modules/getting-started-openshift-common-terms.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-overview.adoc
+
+:_content-type: REFERENCE
+[id="getting-started-openshift-common-terms_{context}"]
```

Comment:
I do really like the idea of defining common terms. I'd actually consider moving this to a whole separate "Glossary" assembly at some point and building it out some.